components/lib/api/PrimeIcons.js

```diff
@@ -1,263 +1,316 @@
-    REPLY: 'pi pi-reply',
-    SAVE: 'pi pi-save',
-    SEARCH: 'pi pi-search',
-    SEARCH_MINUS: 'pi pi-search-minus',
-    SEARCH_PLUS: 'pi pi-search-plus',
-    SEND: 'pi pi-send',
-    SERVER: 'pi pi-server',
-    SHARE_ALT: 'pi pi-share-alt',
-    SHIELD: 'pi pi-shield',
-    SHOPPING_BAG: 'pi pi-shopping-bag',
-    SHOPPING_CART: 'pi pi-shopping-cart',
-    SIGN_IN: 'pi pi-sign-in',
-    SIGN_OUT: 'pi pi-sign-out',
-    SITEMAP: 'pi pi-sitemap',
-    SLACK: 'pi pi-slack',
-    SLIDERS_H: 'pi pi-sliders-h',
-    SLIDERS_V: 'pi pi-sliders-v',
-    SORT: 'pi pi-sort',
-    SORT_ALPHA_DOWN: 'pi pi-sort-alpha-down',
-    SORT_ALPHA_DOWN_ALT: 'pi pi-sort-alpha-down-alt',
-    SORT_ALPHA_UP: 'pi pi-sort-alpha-up',
-    SORT_ALPHA_UP_ALT: 'pi pi-sort-alpha-up-alt',
-    SORT_ALT: 'pi pi-sort-alt',
-    SORT_ALT_SLASH: 'pi pi-sort-alt-slash',
-    SORT_AMOUNT_DOWN: 'pi pi-sort-amount-down',
-    SORT_AMOUNT_DOWN_ALT: 'pi pi-sort-amount-down-alt',
-    SORT_AMOUNT_UP: 'pi pi-sort-amount-up',
-    SORT_AMOUNT_UP_ALT: 'pi pi-sort-amount-up-alt',
-    SORT_DOWN: 'pi pi-sort-down',
-    SORT_NUMERIC_DOWN: 'pi pi-sort-numeric-down',
-    SORT_NUMERIC_DOWN_ALT: 'pi pi-sort-numeric-down-alt',
-    SORT_NUMERIC_UP: 'pi pi-sort-numeric-up',
-    SORT_NUMERIC_UP_ALT: 'pi pi-sort-numeric-up-alt',
-    SORT_UP: 'pi pi-sort-up',
-    SPINNER: 'pi pi-spinner',
-    STAR: 'pi pi-star',
-    STAR_FILL: 'pi pi-star-fill',
-    STEP_BACKWARD: 'pi pi-step-backward',
-    STEP_BACKWARD_ALT: 'pi pi-step-backward-alt',
-    STEP_FORWARD: 'pi pi-step-forward',
-    STEP_FORWARD_ALT: 'pi pi-step-forward-alt',
-    STOP: 'pi pi-stop',
-    STOP_CIRCLE: 'pi pi-stop-circle',
-    STOPWATCH: 'pi pi-stopwatch',
-    SUN: 'pi pi-sun',
-    SYNC: 'pi pi-sync',
-    TABLE: 'pi pi-table',
-    TABLET: 'pi pi-tablet',
-    TAG: 'pi pi-tag',
-    TAGS: 'pi pi-tags',
-    TELEGRAM: 'pi pi-telegram',
-    TH_LARGE: 'pi pi-th-large',
-    THUMBS_DOWN: 'pi pi-thumbs-down',
-    THUMBS_DOWN_FILL: 'pi pi-thumbs-down-fill',
-    THUMBS_UP: 'pi pi-thumbs-up',
-    THUMBS_UP_FILL: 'pi pi-thumbs-up-fill',
-    TICKET: 'pi pi-ticket',
-    TIMES: 'pi pi-times',
-    TIMES_CIRCLE: 'pi pi-times-circle',
-    TRASH: 'pi pi-trash',
-    TRUCK: 'pi pi-truck',
-    TWITTER: 'pi pi-twitter',
-    UNDO: 'pi pi-undo',
-    UNLOCK: 'pi pi-unlock',
-    UPLOAD: 'pi pi-upload',
-    USER: 'pi pi-user',
-    USER_EDIT: 'pi pi-user-edit',
-    USER_MINUS: 'pi pi-user-minus',
-    USER_PLUS: 'pi pi-user-plus',
-    USERS: 'pi pi-users',
-    VERIFIED: 'pi pi-verified',
-    VIDEO: 'pi pi-video',
-    VIMEO: 'pi pi-vimeo',
-    VOLUME_DOWN: 'pi pi-volume-down',
-    VOLUME_OFF: 'pi pi-volume-off',
-    VOLUME_UP: 'pi pi-volume-up',
-    WALLET: 'pi pi-wallet',
-    WHATSAPP: 'pi pi-whatsapp',
-    WIFI: 'pi pi-wifi',
-    WINDOW_MAXIMIZE: 'pi pi-window-maximize',
-    WINDOW_MINIMIZE: 'pi pi-window-minimize',
-    WRENCH: 'pi pi-wrench',
-    YOUTUBE: 'pi pi-youtube'
+    ADDRESS_BOOK: 'pi .pi-address-book',
```

Comment:
Uh, careful here! In the previous version, there was no period before the class name, but now there is.

Reply:
Ahh yeah I need to remove the period.